### DISCUSSION TITLE
Finished blog post about installing Nextcloud with Docker on RHEL

### DISCUSSION THREAD

**Discussion by Verhoeckx** [nextcloud/all-in-one#2169] [Wiki]
Hello Nextcloud AIO developers,

This is just a show-off: I wrote a blog post about how you can install Nextcloud AIO with Docker on Red Hat Enterprise Linux!

You can find that blog post here: 
[https://jeroenverhoeckx.com/install-nextcloud-with-docker.html](https://jeroenverhoeckx.com/install-nextcloud-with-docker.html)

The blog post is already two weeks old, but this is the first time that I share it here.

Let me know what you think about it!

Verhoeckx

**Comment by szaimen** [comment]
Wow, this looks like a pretty complete and well-written guide! Thanks a lot for creating it! :)

I just converted the discussion into a wiki page so that it doesnt get lost over time! :)

**Reply by Verhoeckx** [reply]
Ah, thanks for your complement! I hoped you would like it! 
Hopefully it will help others setting up an instance of Nextcloud AIO!
Best wishes!

**Comment by kri164** [comment]
There's no need to switch firewalld backend to iptables anymore. Im running docker/AIO on firewall+nftables backend several month without any problem.

**Reply by Verhoeckx** [reply]
Do you have some documentation that this has been resolved? An issue report or something?
